test/hack/cleanup/main.go

```diff
@@ -105,6 +132,22 @@ func main() {
 	if err := fireMetric(ctx, cloudWatchClient, "LaunchTemplatesDeleted", float64(deleted)); err != nil {
 		logger.With("name", "LaunchTemplatesDeleted").Errorf("firing metric, %v", err)
 	}
+
+	names = getOldOICD(ctx, iamClient, expirationTime)
+	deleted = 0
+	for i := range names {
```

Comment:
```suggestion
	for i := range arns {
```
If this is ARNs, can we name the variable as such?